lib/bot.js

```diff
@@ -111,6 +111,7 @@ class Bot {
   parseText(message) {
     const text = message.mentions.reduce((content, mention) => (
       content.replace(`<@${mention.id}>`, `@${mention.username}`)
+             .replace(`<@!${mention.id}>`, `@${mention.username}`)
```

Comment:
Ah, I see! Would you mind adding a test as well?
Should be pretty identical to this one: https://github.com/reactiflux/discord-irc/blob/master/test/bot.test.js#L229-L239

(You run the tests with npm test)